frontend-ssr/components/cli_git/GitCli_CC.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { uuid } from '../../services/Utils';
+
+import styles from './GitCli.module.scss';
+
+interface Props {
+  commands: Commands[],
+}
+
+interface State {
+  currentIndex: number,
+  messages: Message[],
+  btnMessage: string,
+}
+
+interface Commands {
+  command: RegExp | string,
+  message: string,
+}
+
+interface Message {
+  text: string,
+  type: MessageType,
+  time: number,
+  id: string,
+}
+
+enum MessageType {
+  COMMAND = 'command',
+  ERROR = 'error',
+  SUCCESS = 'success'
+}
+
+class matchCommandMessageClassComponent extends React.Component<Props, State, Commands> {
```

Comment:
Class components should be `PascalCase` meaning `MatchCommandMessageClassComponent`. Also, why the 3rd param, shouldn't it be just `<Props,State>`?